src/main/java/com/demo/crud/operation/controller/HomeController.java

```diff
@@ -0,0 +1,84 @@
+package com.demo.crud.operation.controller;
+
+import com.demo.crud.operation.beans.Response;
+import com.demo.crud.operation.beans.UserDetails;
+import com.demo.crud.operation.service.DataService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * @author Amit Ojha
+ */
+@RestController
+@RequestMapping("/demo-crud")
+@Api(tags = "User-Controller", description = "User Controller")
+public class HomeController {
+
+    @Autowired
+    private DataService dataService;
+
+    /**
+     * This will return as a list of all user data.
+     *
+     * @return UserDetails
+     */
+    @GetMapping("/all")
+    @ApiOperation(value = "Get all user details")
+    public ResponseEntity<Response> getAllUsersDetails() {
```

Comment:
In all api mapping you are returning Ok status (200 status code) but that should not be a case i think , we should return this status code based on the usage like its server error or new user is added then 201 status or Created

Reply:
200 ok status is used for successfully working of API and it gives me response no error occur,so i use this.